case-studies/simple/draft1.md

```diff
@@ -0,0 +1,215 @@
+
+---
+
+The 2019 monsoon was upon India and Akshay was on his way to Punjab for a field visit. It was his first time seeing the clinics in person and he wasn't entirely sure what to expect. Superhuman nurses? Would the police chase a criminal through the hospital again? Just how remote were these clinics, anyway?
+
+He was awestruck. "These places are so remote that the word 'remote' doesn't do them justice," he said. "We would often drive around for what felt like an eternity to find these clinics. When we finally found them, my first impulse was always to add them to Google Maps and OpenStreetMap so other people could actually find them. It was so satisfying when we would actually locate the facility... and then it was always surprising to see just how many people were inside."
+
+"These are USADs," said Akshay. "Urban Slum Area Dispensaries. Some of them are just a box. Often we would go inside to find fifty to a hundred patients... in an area smaller than my house. And yet, there is the nurse. Calm. Relaxed. Effective. I think they have to remain mentally resolute just to stay sane."
+
+And so the stories were true. The superhuman nurse, with her seemingly infinite patience and a resolve undaunted by the daily obstacles present in a clinic where most of us wouldn't survive a single workday. No matter who you are, she's here to help you.
+
+"My Mom used to work as a doctor in one of these remote village dispensaries when I was a kid," said Akshay. "The clinic was sort of forked off from a hospital where she'd worked and she went along to join the clinic. I remember visiting that clinic. It was very much like these dispensaries. After the day's dispensary work, she would teach English to the local kids for an hour before coming home to the city." He paused. "Yeah. It was just like this."
+
+Every day, the team's field visit to the clinic would begin at 4:30 AM. Wake up, drink tea, get ready, leave by 6:00 AM, arrive by 7:00 AM. One such day, despite a forecast for clear skies, it rained. On the drive to the clinic Akshay looked out the window at Punjab's fields stretching to the horizon, sparkling emerald and jade under fresh rain, smiling back at him. The clinic was silent. Patients don't come during a monsoon rain. Shortly after the team arrived, there was a small ruckus when a snake slithered into the quiet clinic and the staff struggled to get rid of it. Once the clinic was free of snakes, Deepa, Akshay, the designers, the nurses, the doctors --- everyone sat together having tea and biscuits, enjoying the company of the others for a few tranquil hours until the skies cleared at 10:00 AM and the day's first patients began to trickle in. In contrast to the clinic's normal, non-stop workday this was a rare opportunity for everyone to share their uncluttered humanity.
```

Comment:
No Deepa in this story.

Reply:
Oh, sorry. Deepa had mentioned that she had sat down for tea and Parle-G with the nurses once and I naturally assumed this was the same time. Who was all there with you for this event? (You can poke me on Slack for this, if you'd rather.)